src/simplewallet/simplewallet.h

```diff
@@ -363,5 +364,44 @@ namespace cryptonote
     bool m_auto_refresh_refreshing;
     std::atomic<bool> m_in_manual_refresh;
     uint32_t m_current_subaddress_account;
+    
+    // MMS
+    mms::message_store& get_message_store() const { return m_wallet->get_message_store(); };
```

Comment:
Not sure what you mean. My intention here: The function itself is `const` because the `simple_wallet` object does not get modified by calling it. The returned `message_store` however should be open to modification and does get modified indeed in many cases afterwards.

Reply:
That reasoning means some function can use a const wallet passed and still modify the message store it contains without const casts or the like.

Reply:
I duplicated that function, with a completely non-const version and a const/const version.  I think with this I have all bases covered.